.github/workflows/build-workshop-site.yml

```diff
@@ -11,7 +11,8 @@ jobs:
         uses: actions/checkout@v2
```

Comment:
@v3 is the current now. May as well update all the versions if we're touching workflows :)